plugins/modules/authorized_key.py

```diff
@@ -28,8 +28,7 @@
     required: true
   path:
     description:
-      - Alternate path to the authorized_keys file.
-      - When unset, this value defaults to I(~/.ssh/authorized_keys).
```

Comment:
You can remove this line, but you still need to mention the default path somewhere.

Reply:
Thanks, you still need to mention that `I(~/.ssh/authorized_keys)` is the default value.